app/assets/javascripts/discourse/app/components/user-menu/bookmark-reminder-notification-item.js

```diff
@@ -0,0 +1,17 @@
+import UserMenuNotificationItem from "discourse/components/user-menu/notification-item";
+import I18n from "I18n";
+
+export default class UserMenuBookmarkReminderNotificationItem extends UserMenuNotificationItem {
+  get linkTitle() {
+    if (this.notification.data.bookmark_name) {
```

Comment:
Can `notification.data.bookmark_name` ever be blank for a bookmark reminder notification?

Reply:
Yes, `bookmark_name` comes from this input field in the bookmark creation modal:

<img width=400 src="https://user-images.githubusercontent.com/17474474/180120818-0d4a9dd5-7aea-4b87-ba8f-185a26df10e2.png">

That input field is optional and when it's left blank `bookmark_name` is nil.

Reply:
:+1: